pages/bare_metal_cloud/dedicated_servers/partitioning_ovh/guide.en-gb.md

```diff
@@ -210,30 +401,116 @@ In order to make the OVHcloud API as simple as possible, customers must define a
 |RAID 7|raidz3|4|3|7|
 |RAID 10|❌|4|0|4|
 
-### Partitioning Backend
+#### API & Partitioning
+
+Example of OS installation with a custom partitioning layout:
+
+> [!api]
+>
+> @api {v1} /dedicated/server POST /dedicated/server/{serviceName}/reinstall
+>
+
+```json
+{
+    "operatingSystem": "debian12_64",
+    "storage": [
+        {
+            "partitioning": {
+                "disks": 2,
+                "layout": [
+                    {
+                        "fileSystem": "ext4",
+                        "mountPoint": "/boot",
+                        "size": 1000,
+                        "raidLevel": 0
+                    },
+                    {
+                        "fileSystem": "ext4",
+                        "mountPoint": "/",
+                        "size": 0
+                    }
+                ]
+            }
+        }
+    ]
+}
+```
+
+`size` 0 means that the partition will fill all the remaining space on the involved disks.
 
-#### Architecture
+> [!primary]
+>
+> Up to 1 partition can be configured to fill the disk (size 0).
+>
 
-There is an abstraction layer between the partitioning description on the OVHcloud API (that is quite minimalist) and the real partitioning (that can be very complex).
+<br />
```

Comment:
Not really needed here

Reply:
Not sure where this is exactly, but after checking locally, `<br />` between 2 primary/warning stuffs is mandatory to prevent issue that will cut the logo of the second one...